### ISSUE TITLE
Cannot tab into the decision table header

### ISSUE THREAD

**Issue by barmac** [bpmn-io/dmn-js#845]
### Describe the Bug

The table header is unreachable with a keyboard, so it requires mouse doubleclick to modify.

### Steps to Reproduce

Try to tab to the input/output header.

### Expected Behavior

Table header should be focussable and user should be able to open it. It can open automatically or via Enter.

### Environment

Please complete the following information:

 - Browser: [e.g. IE 11, Chrome 69]
 - OS: [e.g. Windows 7]
 - Library version: [e.g. 2.0.0]

---

Child of https://github.com/bpmn-io/internal-docs/issues/909

**Comment by barmac** [comment]
I considered adding an edit button like we already have in the boxed expression, but it would hide the "move column" handler:

<img width="199" alt="image" src="https://github.com/bpmn-io/dmn-js/assets/28307541/31c57cd9-4bcc-4230-bd51-b5ba4db81eff">
